### PR TITLE
fix: rsync fewer files

### PR DESCRIPTION
**Pull request by larseggert**
It's still super slow, however.

### CODE REVIEW

File: docker/scripts/app-rsync-extras.sh [hunk 1]
```diff
@@ -33,33 +33,66 @@ done
...
 mkdir -p "$dest"
 echo "Fetching $dest ..."
-rsync -auz ${PROGRESS:+--info=progress2} rsync.ietf.org::id-archive/ $dest/
+rsync $OPTS rsync.ietf.org::id-archive/ $dest/
 
 dest="$DEST_ROOT/www6s/proceedings"
 mkdir -p "$dest"
 echo "Fetching $dest ..."
-rsync -auz ${PROGRESS:+--info=progress2} rsync.ietf.org::proceedings/ $dest/
+rsync $OPTS rsync.ietf.org::proceedings/ $dest/
```
Comment (rjsparks) [inline docker/scripts/app-rsync-extras.sh:96]: Do we really need to sync all the proceedings?

**Review by rjsparks — APPROVED** [review]

File: docker/scripts/app-rsync-extras.sh (hunk 1 above)
Comment (larseggert) [reply docker/scripts/app-rsync-extras.sh:96]: I don't think so. Do we need _any_?

Comment (rjsparks) [reply docker/scripts/app-rsync-extras.sh:96]: yes, it's where the minutes/agendas/slides are currently stored, so getting the most recent and upcoming meetings would likely be useful by default.

(We need to change where these are stored on disk in the long run)

Comment (larseggert) [reply docker/scripts/app-rsync-extras.sh:96]: So one quick fix would be not not rsync proceedings of IETFs <100 (which is possible using the limited patterns that rsync allows) - should I add that?

Comment (rjsparks) [reply docker/scripts/app-rsync-extras.sh:96]: Try it and look at the size. I think you could get 11* easily too?

**Pull request merged**